test/irb/test_completion.rb

```diff
@@ -14,11 +14,19 @@ def setup
     class TestMethodCompletion < TestCompletion
       def test_complete_string
         assert_include(IRB::InputCompletor.retrieve_completion_data("'foo'.up", bind: binding), "'foo'.upcase")
+        assert_not_include(IRB::InputCompletor.retrieve_completion_data("'foo'.up", bind: binding), "'foo'.update")
```

Comment:
I think the test was not working because completion will fallback to listup all methods using ObjectSpace search.
all methods including upcase will be shown, and the test in L16 passes whether string completion is implemented or not.
I add this `assert_not_include` to check that it does not include `Hash#update`